packages/lib/ren/src/index.ts

```diff
@@ -157,6 +159,18 @@ export default class RenJS {
                           | RenNetworkDetails,
                       this._logger,
                   );
+        if (typeof providerOrNetwork === "string") {
+            this.networkName = providerOrNetwork;
+        } else if (
+            providerOrNetwork &&
+            (providerOrNetwork as AbstractRenVMProvider).sendMessage
+        ) {
+            this.networkName = providerOrNetwork as AbstractRenVMProvider;
```

Comment:
This isn't building for me:
```
Type 'AbstractRenVMProvider<{}, {}>' is not assignable to type 'string'.
```

Reply:
Ah crud, these changes weren't supposed to land here, will revert